### DISCUSSION TITLE
Time between new release and Microsoft Store publishing

### DISCUSSION THREAD

**Discussion by Solomon1732** [microsoft/terminal#17975] [Q&A] [answered]
Since a release is pushed in GitHub, how long does it take for it to reach Microsoft Store? I installed the Terminal through it. I see the release-before-last, [v1.21.2361.0](https://github.com/microsoft/terminal/releases/tag/v1.21.2361.0), was release on August 27th. I'm still using [v1.20.11781.0](https://github.com/microsoft/terminal/releases/tag/v1.20.11781.0) from June 28th. It is now September 29th, and my Terminal didn't yet update to it (I'm ignoring the latest release from two days ago). Just to check, I even uninstalled it, then reinstalled it. Still no change. 

Am I missing something?

**Comment by j4james** [comment, marked as answer]
@Solomon1732 It's been mentioned that v1.21 is being rolled out slower than usual because of an IME issue that affects some versions of Windows (see https://github.com/microsoft/terminal/issues/17952#issuecomment-2367718562). I suspect that may be why you aren't seeing the update in Microsoft Store yet.
